### DISCUSSION TITLE
3.2.0

### DISCUSSION THREAD

**Discussion by LabhanshAgrawal** [vercel/hyper#6286] [Announcements]
V3.2.0

|   OS    | Installer |                                                                                                 |                                                                                                  |
| :-----: | :-------: | :---------------------------------------------------------------------------------------------: | :----------------------------------------------------------------------------------------------: |
|   Mac   |    dmg    | [x64](https://github.com/vercel/hyper/releases/download/v3.2.0/Hyper-3.2.0-mac-x64.dmg) (Intel) | [arm64](https://github.com/vercel/hyper/releases/download/v3.2.0/Hyper-3.2.0-mac-arm64.dmg) (M1) |
|  Linux  |    deb    |      [x64](https://github.com/vercel/hyper/releases/download/v3.2.0/hyper_3.2.0_amd64.deb)      |     [arm64](https://github.com/vercel/hyper/releases/download/v3.2.0/hyper_3.2.0_arm64.deb)      |
|         |    rpm    |     [x64](https://github.com/vercel/hyper/releases/download/v3.2.0/hyper-3.2.0.x86_64.rpm)      |    [arm64](https://github.com/vercel/hyper/releases/download/v3.2.0/hyper-3.2.0.aarch64.rpm)     |
|         | AppImage  |      [x64](https://github.com/vercel/hyper/releases/download/v3.2.0/Hyper-3.2.0.AppImage)       |   [arm64](https://github.com/vercel/hyper/releases/download/v3.2.0/Hyper-3.2.0-arm64.AppImage)   |
|         |   snap    |     [x64](https://github.com/vercel/hyper/releases/download/v3.2.0/hyper_3.2.0_amd64.snap)      |                                                                                                  |
| Windows |    exe    |      [x64](https://github.com/vercel/hyper/releases/download/v3.2.0/Hyper-Setup-3.2.0.exe)      |                                                                                                  |

## Changelog
- Add CWD preservation (#6225)
- Fix window contents not clearing properly (#6285)
- Focus terminal on closing search box (#6230)
- Update electron to v16 (#6280)
- Use electron's arm64 detection (#6281)
- Fix build-id conflicts with other electron rpm packages (#6232)

**Full Changelog**: https://github.com/vercel/hyper/compare/v3.1.5...v3.2.0

<hr /><em>This discussion was created from the release <a href='https://github.com/vercel/hyper/releases/tag/v3.2.0'>3.2.0</a>.</em>

**Comment by alecgibson** [comment]
For anyone trying to put CWD behaviour back to how it was, update your `.hyper.js`:

```js
module.exports = {
  config: {
    preserveCWD: false,
  },
};
```
